drivers/cst816s/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 Inria
```

Comment:
Should this be 2022? The commit is from 2022 :)

Reply:
Fixed, I'll test this thursday on my pinetime